src/components/Chat/MessageWithImage.js

```diff
@@ -250,7 +250,7 @@ const styles = StyleSheet.create({
     marginHorizontal: 20
   },
   containerChat: (isMe) => ({
-    backgroundColor: isMe ? COLORS.halfBaked : COLORS.lightgrey,
+    backgroundColor: isMe ? COLORS.halfBaked : COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color of the chat container has been changed based on whether the message is sent by the user or not. The new colors should be reviewed in the context of the overall UI design and the text color to ensure that there is enough contrast for readability.
```diff
-    backgroundColor: isMe ? COLORS.halfBaked : COLORS.lightgrey,
+    backgroundColor: isMe ? COLORS.halfBaked : COLORS.gray100,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->